### PR TITLE
Added documentation for the ELV PCA 301 smart emeter switch

### PR DESCRIPTION
**Pull request by majuss**
**Description:**
Adds the documentation for the ELV PCA 301 smart emeter switch running at 868 MHz.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#23300

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/switch.pca.markdown [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+switch:
+  - platform: pca
+    device: SERIAL_PORT
+```
+
+This platform will add all PCA 301 switches which are in range. You can read the total used energy in KWh and the current power in Watt.
+
+{% configuration %}
+device:
+  description: The path to you serial tty. Get it via: `ls /dev/tty*`.
```
Comment (klaasnicolaas) [inline source/_components/switch.pca.markdown]:
```suggestion
  description: "The path to you serial tty. Get it via: `ls /dev/tty*`."
```

**Comment by majuss** [comment]
Had in typo in there, now it should work.

File: source/_components/switch.pca.markdown [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+title: "ELV PCA 301 Switch"
+description: "Instructions on how to integrate ELV PCA 301 switches into Home Assistant."
+date: 2019-02-24 18:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: elv.png
+ha_category: Switch
+ha_iot_class: "Local Polling"
```
Comment (klaasnicolaas) [inline source/_components/switch.pca.markdown]:
```suggestion
ha_iot_class: Local Polling
```

File: source/_components/switch.pca.markdown [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+ha_release: "0.90"
```
Comment (klaasnicolaas) [inline source/_components/switch.pca.markdown]:
```suggestion
ha_release: 0.91
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/switch.pca.markdown [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+```yaml
+# Example configuration.yaml entry
+switch:
+  - platform: pca
+    device: SERIAL_PORT
+```
+
+This platform will add all PCA 301 switches which are in range. You can read the total used energy in KWh and the current power in Watt.
+
+{% configuration %}
```
Comment (klaasnicolaas) [inline source/_components/switch.pca.markdown:31]: I see that there is a config variable for `name`, you can add this here.

File: source/_components/switch.pca.markdown [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+sharing: true
+footer: true
+logo: elv.png
+ha_category: Switch
+ha_iot_class: Local Polling
+ha_release: 0.91
+---
+
+The `pca` switch platform allows you to control the state of your [ELV PCA 301 smart switch](https://www.elv.de/funkschaltsteckdose-fuer-energiekostenmonitor-pca-301.html). You need a 868 MHz interface like the [JeeLink](https://www.digitalsmarties.net/products/jeelink) flashed with the [pca-hex firmware](https://github.com/mhop/fhem-mirror/blob/master/fhem/FHEM/firmware/JeeLink_PCA301.hex).
+
```
Comment (klaasnicolaas) [inline source/_components/switch.pca.markdown]: One whitespace to much 😉

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]
After this changes, we can approve it 🎉

**Review by klaasnicolaas — DISMISSED** [review]
✅ Can be merged after parent PR got merged 👍

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thanks, @majuss! 👍

**Comment by majuss** [comment]
It's always a pleasure to work with you guys and to expand the supported platforms! :)

**Comment by majuss** [comment]
Hm something is off, maybe some pca/elv mixup, I can't reach the documentation.

**Comment by klaasnicolaas** [comment]
https://www.home-assistant.io/components/switch.pca/

Maybe it was better if the platform was called `elf` and the linke ended with /elv/

**Pull request merged**